kafka-rest-common/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -65,6 +65,13 @@
       + " hostname is used";
   public static final String HOST_NAME_DEFAULT = "";
 
+  public static final String ADVERTISED_LISTENERS_CONFIG = "advertised.listeners";
```

Comment:
I have not spent any time thinking about the advantages of including absolute URLs in responses, but for your consideration, want to note that requiring this additional config seems like a material downside.

Reply:
The configuration is not required. See UrlFactoryImpl for how the absolute URLs are generated. This configuration is to allow for generating URLs using a LB.